docs/resources/rest_managed.md

```diff
@@ -109,12 +109,20 @@ An existing object can be [imported](https://www.terraform.io/docs/import/index.
 terraform import aci_rest_managed.example_tenant uni/tn-{name}
 ```
 
+!> The use of the colon-separated format to import multiple child resources will be deprecated in the next release.
+
```

Comment:
Should we remove the deprecated example from the documentation, to force people to use the new example?